spec/amp-versioning-policy.md

```diff
@@ -64,6 +64,8 @@ The process for deprecations is similar to the [process for significant changes]
 - Give developers at least 6 weeks after your deprecation warning is in production to apply changes before removing the feature.
 - If you did not include removal in your I2D, file a separate Intent-to-Remove (I2R) when you are ready to remove the deprecated feature.
 
+The given component or integration may be neutered immediately in such a way that usage in a document no longer has side effects (e.g. the underlying backend has been turned down), but validation does not break.
```

Comment:
I'd start this with "If an underlying backend service has itself been turned down, then the given…"

Reply:
does that revert your previous comment? :-) which was try to generalize the criteria. 

frankly, I like your new suggestion better as it's easier to follow was a guideline.

Reply:
Updated. PTAL